Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM ubuntu:18.04
 ARG MODE=default
 RUN --mount=target=/polycube cp -r /polycube /tmp/polycube && \
 cd /tmp/polycube && \
-SUDO="" WORKDIR="/tmp/dev" ./scripts/install.sh $MODE && \
+SUDO="" USER="root" WORKDIR="/tmp/dev" ./scripts/install.sh $MODE && \
```

Comment:
Do we need to specify the user as "root"? If I'm not mistaken as long as the user have sudo privilege, it should be fine. 
@mauriciovasquezbernal any thoughts?

Reply:
the problem here is that without setting the USER environment variable, docker building will fail in this line silently, 
install(CODE
"execute_process( COMMAND /bin/sh -c \"mkdir -p /var/log/polycube && chown $ENV{USER} /var/log/polycube \")")